Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp

```diff
@@ -21,8 +21,7 @@ namespace PhysX
             serializeContext->Class<EditorStaticRigidBodyComponent, AzToolsFramework::Components::EditorComponentBase>()
                 ->Version(1);
 
-            AZ::EditContext* editContext = serializeContext->GetEditContext();
-            if (editContext)
+            if (auto* editContext = serializeContext->GetEditContext())
```

Comment:
uber nit: (could do the same above for the `serializeContext` if you haven't started AR yet)

Reply:
It goes to a staging branch, there is no AR involved yet. Change done.

Reply:
Awesome thanks!